admin-app/dictionary/management/commands/import_words.py

```
@@ -0,0 +1,45 @@
+
+            imported_count = 0
+            for item in words_data:
+                word = Word.objects.create(
+                    term=item['term'][:200],
+                    description=item.get('description', ''),
+                    source=item.get('sources', '')
+                )
+
+                if item.get('tags'):
+                    for tag_name in item['tags']:
+                        tag, tag_created = Tag.objects.get_or_create(name=tag_name)
```

Ruff (0.15.2)

[warning] Unpacked variable `tag_created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)